### ISSUE TITLE
Port py_vnr v2.1.0 changes to fwk_voice

### ISSUE THREAD

**Issue by shuchitak** [xmos/fwk_voice#336]

**Comment by shuchitak** [comment]
Results:

Current head of develop of Avona
[develop_results_Avona_prev_arch_xcore.csv](https://github.com/xmos/fwk_voice/files/9139097/develop_results_Avona_prev_arch_xcore.csv)
[develop_results_Avona_alt_arch_xcore.csv](https://github.com/xmos/fwk_voice/files/9139098/develop_results_Avona_alt_arch_xcore.csv)

With py_vnr v2.1.0 ported
[vnr_v2_1_0_results_Avona_prev_arch_xcore.csv](https://github.com/xmos/fwk_voice/files/9139101/vnr_v2_1_0_results_Avona_prev_arch_xcore.csv)
[vnr_v2_1_0_results_Avona_alt_arch_xcore.csv](https://github.com/xmos/fwk_voice/files/9139102/vnr_v2_1_0_results_Avona_alt_arch_xcore.csv)

With py_vnr v2.1.0 and the highpass filter (which is disabled by default) on MEL output enabled
[hpf_vnr_v2_1_0_results_Avona_prev_arch_xcore.csv](https://github.com/xmos/fwk_voice/files/9139110/hpf_vnr_v2_1_0_results_Avona_prev_arch_xcore.csv)
[hpf_vnr_v2_1_0_results_Avona_alt_arch_xcore.csv](https://github.com/xmos/fwk_voice/files/9139111/hpf_vnr_v2_1_0_results_Avona_alt_arch_xcore.csv)

**Comment by shuchitak** [comment]
py_vnr v2.1.0 vs head of Avona develop:

Prev-arch
![vnr_v2_1_0_prev_arch](https://user-images.githubusercontent.com/38428600/179708277-84c29fca-30e9-44c5-980f-f79f8fd22a03.png)
[vnr_v2_1_0_prev_arch.csv](https://github.com/xmos/fwk_voice/files/9139143/vnr_v2_1_0_prev_arch.csv)

Alt-arch
![vnr_v2_1_0_alt_arch](https://user-images.githubusercontent.com/38428600/179708285-77d94c84-af4e-442c-adbc-9ef484874bf3.png)
[vnr_v2_1_0_alt_arch.csv](https://github.com/xmos/fwk_voice/files/9139144/vnr_v2_1_0_alt_arch.csv)

**Comment by shuchitak** [comment]
py_vnr v2.1.0, with and without hpf

Prev-arch
![hpf_vnr_v2_1_0_prev_arch](https://user-images.githubusercontent.com/38428600/179708534-1f10388a-94b7-4d7c-b411-73eda73d1fe0.png)
[hpf_vnr_v2_1_0_prev_arch.csv](https://github.com/xmos/fwk_voice/files/9139150/hpf_vnr_v2_1_0_prev_arch.csv)
![hpf_vnr_v2_1_0_alt_arch](https://user-images.githubusercontent.com/38428600/179708546-6cfd4550-91dc-49c1-8eb2-c33fb76aeff1.png)

Alt-arch
[hpf_vnr_v2_1_0_alt_arch.csv](https://github.com/xmos/fwk_voice/files/9139151/hpf_vnr_v2_1_0_alt_arch.csv)

**Comment by andrewdewhurst** [comment]
Agreed with Shuchita, Pavel, Alan and Thanasis that this is complete.
